src/hypothesis/strategies.py

```diff
@@ -1588,6 +1576,90 @@ def label(self):
     accept.__module__ = f.__module__
     return accept
 
+@composite
+def complex_numbers(
```

Comment:
I moved it because `NameError: name 'composite' is not defined` where it was -- is there some way to declare a decorator before definition?  Or we could move `composite` up the top of the file?

Reply:
Oh, of course.  That's a good reason; just keep it in the new location then.